huxley/www/js/components/AdvisorFeedbackView.js

```diff
@@ -0,0 +1,204 @@
+/**
+* Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+* Use of this source code is governed by a BSD License (see LICENSE).
++*/
+
+'use strict';
+
+var React = require('react');
+var ReactRouter = require('react-router');
+
+var AssignmentActions = require('actions/AssignmentActions');
+var AssignmentStore = require('stores/AssignmentStore');
+var Button = require('components/Button');
+var CommitteeStore = require('stores/CommitteeStore');
+var ConferenceContext = require('components/ConferenceContext');
+var CountryStore = require('stores/CountryStore');
+var CurrentUserStore = require('stores/CurrentUserStore');
+var CurrentUserActions = require('actions/CurrentUserActions');
+var DelegateActions = require('actions/DelegateActions');
+var DelegateSelect = require('components/DelegateSelect');
+var DelegateStore = require('stores/DelegateStore');
+var InnerView = require('components/InnerView');
+var ServerAPI = require('lib/ServerAPI');
+
+var AdvisorFeedbackView = React.createClass({
+  mixins: [
+    ReactRouter.History,
+  ],
+
+  contextTypes: {
+    conference: React.PropTypes.shape(ConferenceContext)
+  },
+
+  getInitialState: function() {
+    var schoolID = CurrentUserStore.getCurrentUser().school.id;
+    var delegates = DelegateStore.getSchoolDelegates(schoolID);
+    var assigned = this.prepareAssignedDelegates(delegates);
+    return {
+      assigned: assigned,
+      assignments: AssignmentStore.getSchoolAssignments(schoolID).filter(assignment => !assignment.rejected),
+      committees: CommitteeStore.getCommittees(),
+      countries: CountryStore.getCountries(),
```

Comment:
So you don't ever end up actually using this. However it might make more sense to instead label each row by country instead of by names of delegate. It will be more reflective of our models later on down the road, cause we should transition holding the attendance and summary states away from the Delegate model and onto the Assignment model or its own model as @kmeht suggested. But that's for a later PR